src/indra/fetch/cds.py

```
@@ -56,314 +63,465 @@ def last_date_of_cds_data(suppress_output=True):
+        "--dir", output_dir,
+        "--max-concurrent-downloads", str(max_connections),
+        "--max-connection-per-server", str(max_connections),
+        "--split", str(max_connections),
+        "--min-split-size", "1M",
+        "--continue=true",
+        "--auto-file-renaming=false",
+        "--allow-overwrite=true",
+        "--console-log-level=warn",
+    ]
+    logger.info("Launching aria2c: %s", " ".join(cmd))
+    proc = subprocess.run(cmd, capture_output=True, text=True)
```

Ruff (0.15.7)

[error] `subprocess` call: check for execution of untrusted input

(S603)